php/8.4-unit/unit/docker-entrypoint.sh

```
@@ -0,0 +1,82 @@
+            for f in $(/usr/bin/find /docker-entrypoint.d/ \( -type l -o -type f \) -name "*.sh"); do
+                echo "$0: Launching $f";
+                "$f"
+            done
+
+            # warn on filetypes we don't know what to do with
+            for f in $(/usr/bin/find /docker-entrypoint.d/ \( -type l -o -type f \) -not -name "*.sh" -not -name "*.json" -not -name "*.pem"); do
+                echo "$0: Ignoring $f";
+            done
+
+            echo "$0: Stopping Unit daemon after initial configuration..."
+            kill -TERM $(/bin/cat "$UNIT_PID")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)